### PR TITLE
Cleanup github actions documentation

### PR DESCRIPTION
**Pull request by soerenmartius**
- Cleanup the workflows
- Focus on Terramate Cloud
- Add installation details for the GitHub App and for configuring the GitHub Trust in the Terramate Cloud Organization

### CODE REVIEW

File: cli/automation/github-actions/index.md [hunk 1]
```diff
@@ -40,20 +70,18 @@ Ensure that you are not using the `terraform` wrapper when using the Terraform S
...
     terraform_wrapper: false
-
 ```
-
 :::
 
 To install Terraform using the [`hashicorp/setup-terraform`](https://github.com/hashicorp/setup-terraform) GitHub Action, you must disable the included wrapper.
 
-The `terraform` wrapper script sets up outputs for follow-up GitHub Actions steps. The recommended method of using the [`terramate-io/terramate-action`](https://github.com/terramate-io/terramate-action) is through its supported wrapper, which shares outputs and exit codes for subsequent actions.
+The `terraform` wrapper script sets up outputs for follow-up GitHub Actions steps. The recommended method of using the [`terramate-io/terramate-action`](https://github.com/terramate-io/terramate-action) is through its supported wrapper, which shares outputs and exit codes for subsequent actions.
```
Comment (i4ki) [inline cli/automation/github-actions/index.md:78]:
```suggestion
The `terraform` wrapper script sets up outputs for follow-up GitHub Actions steps. The recommended method of using the [`terramate-io/terramate-action`](https://github.com/terramate-io/terramate-action) is through its own wrapper, which shares outputs and exit codes for subsequent actions.
```
I think it's better to understand this way but I could be wrong.

File: cli/automation/github-actions/preview-workflow.md [hunk 2]
```diff
@@ -7,26 +7,20 @@ description: Learn how to use Terramate to configure custom GitOps workflows to
 
 The following workflows are blueprints and need some adjustments to work for you.
 
+Note:
+<!-- insert a quick video or pictues describing PR-level -->
+<!-- plan sanitation -->
```
Comment (i4ki) [inline cli/automation/github-actions/preview-workflow.md:12]:
```suggestion
<!-- insert a quick video or pictures describing PR-level -->
<!-- plan sanitization -->
```
:stuck_out_tongue:

**Review by i4ki — APPROVED** [review]
LGTM but left some comments that you can ignore

File: cli/automation/github-actions/index.md [hunk 3]
```diff
@@ -15,10 +15,40 @@ at [terramate-quickstart-aws](https://github.com/terramate-io/terramate-quicksta
...
+
+![GitHub App Pull Request Previews](../../assets/automation/pull-request-preview.png "GitHub App Pull Request Previews")
+
+### Configure GitHub Trust in your Terramate Cloud Organization
+
+To allow Terramate CLI to sync data from GitHub Actions to Terramate Cloud, setting up OpenID Connect (OIDC) is required.
+For that, you must configure the [GitHub Trust](../../../cloud/organization/settings.md#general-settings) in the settings
+page of your Terramate Cloud Organization. This allows you to establish a trust relationship with GitHub Actions, which is
+required to sync data from GitHub, and specify whether to sync the entire organization or a specific repository with
+Terramate Cloud.
```
Comment (AnnuCode) [inline cli/automation/github-actions/index.md:50]:
```suggestion
To sync data from GitHub Actions to Terramate Cloud using the Terramate CLI, set up OpenID Connect (OIDC). Configure the [GitHub Trust](../../../cloud/organization/settings.md#general-settings) in your Terramate Cloud Organization settings. This establishes a trust relationship with GitHub Actions, allowing you to sync data from GitHub. You can sync the entire organization or a specific repository with Terramate Cloud.
```

Comment (AnnuCode) [reply cli/automation/github-actions/index.md:50]: optional suggestion..

**Pull request merged**